pytest-embedded-idf/pytest_embedded_idf/unity_tester.py

```diff
@@ -440,6 +469,7 @@ def run_all_single_board_cases(
             run_ignore_cases: Whether to run ignored test cases or not.
             name: test case name or a list of test case names to run.
             attributes: Dictionary of attributes to filter and run test cases.
+            dry_run: If True, then just show a list of test case names without running them.
```

Comment:
```suggestion
            dry_run: If True, then just show a list of test case names without running them. (please set the logging level as INFO to see them)
```

otherwise the default `WARNING` level won't show these `logging.info` lines